### DISCUSSION TITLE
Cannot add property TiandituImageryProvider, object is not extensible     at instance.createCesiumObject

### DISCUSSION THREAD

**Discussion by whyds000** [zouyaoji/vue-cesium#226] [Q&A]
我在vue3页面中加载天地图影像，没有任何问题，当我把这个页面当做组件传入其他页面时，加载天地图影像报错，各位大佬，请问时什么原因呢

**Comment by whyds000** [comment]
我一个页面同时引入openlayers组件和vue-cesium封装的组件，这时候vue-cesium加载天地图就会报错，只要引入了openlayers组件就会报错，不管是否在tempalte里使用了引入的组件

**Reply by wenjing0213** [reply]
您好，我也遇到了相同的报错，想请问一下您找到解决问题的方法了吗？

**Comment by zouyaoji** [comment]
组件要放到 vc-viewer下面哈。
最好是提供一个复现例子我空了看看。
